config/codon.config

```diff
@@ -14,7 +14,12 @@ workDir = params.workdir
 
 process {
 
-    withName: ALIGNMENT {
+    withName: ALIGNMENT_BAM {
+        cpus = 16
+        memory     = { 50.GB * task.attempt }
+        maxRetries    = 5
```

Comment:
Loads of retries, we should capture the stats from successful runs to tweak this a bit. As that many retries will bloat the nf workdir